Qiqqa/Common/Configuration/WebsiteAccess.cs

```diff
@@ -58,8 +58,6 @@ private static bool ValidateRemoteCertificate(object sender, X509Certificate cer
         public static readonly string Url_LuceneQuerySyntax = @"http://lucene.apache.org/core/old_versioned_docs/versions/2_9_1/queryparsersyntax.html";
         public static readonly string Url_GoogleScholarBibTeXPreferences = @"http://scholar.google.com/scholar_setprefs?scis=yes&scisf=4";
 
-        public static readonly string Url_ChatAPI4Qiqqa = @"http://chatapi.qiqqa.com:8080";

```

Comment:
That one wasn't working anyway, so this is good.

While I would like to keep the number of communication channels low, there is a good reason for having a chat-like channel for *users*. While I might not be the person to "man" that kind of support desk channel, any suggestions while you're at it? 
(Gitter is an option, but there's sure to  be others. As long as it's spam-resistant -- given the !@#$%^& that's happened to the GetSatisfaction Qiqqa forum :-( )

Not to be included in the Qiqqa software, but a link to open a browser from Qiqqa to such a support/discussion channel would be good IMO.

Reply:
I am not a fan of Gitter, no login with password there. From user perspective making github.com account doesn't sound good either.
Getsatisfaction doesn't look too bad, if moderation would be possible (to delete spam etc).
IRC is fine, but is missing threads. E-mail in google groups is slightly better.
I think there is no perfect free solution.

Reply:
About Gitter: yup, feel the same. A lot of folks are using it, but it's too chaotic for my tastes TBH: no topic chains. (Same goes for IRC: I never really got into that one either.)

About GetSatisfaction: I'll nag Jimme again to give me admin access there so the link spammers can be kicked off. (Got no access rights beyond standard user there ATM)

I have set google groups several times before, and it works, still better than web forums (because it's push to single reception point which reduces the number of web spots you have to visit vs.  web forum *pull* where you have to visit the forum to see everything that's going on); filtering at high volumes is a challenge, though.

In my opinion getting a github account is about the same hurdle for a user as subscribing to a web forum (create account, join) or google group (iff you don't have a google account yet / google-recognized account, but, granted, that's a bit like not being on Facebook these days 😉 ... my kind of people 😈 

---

While many folks still believe Github is for IT/programmers, there's all this:

- https://readwrite.com/2013/11/08/seven-ways-to-use-github-that-arent-coding/
- https://medium.com/@jjmerelo/top-ten-reasons-github-is-a-great-tool-for-creative-writers-d0e8b27de71d
- https://github.com/tvanantwerp/github-for-non-programmers --> [http://github-for-non-programmers.跳.net/](http://github-for-non-programmers.xn--vt3a.net/)

but there remains some public perception work to be done before we all would agree getting a github is on the same level of effort as getting a Facebook account.

Thanks for the feedback and your work on Qiqqa.

Reply:
> 
> 
> I am not a fan of Gitter, no login with password there. From user perspective making github.com account doesn't sound good either.
> Getsatisfaction doesn't look too bad, if moderation would be possible (to delete spam etc).
> IRC is fine, but is missing threads. E-mail in google groups is slightly better.
> I think there is no perfect free solution.

From user perspective making github.com account doesn't sound good for _commercial packages_, I think it would be more or less OK for an academic oriented open-source software. More often than not, they will have their own repository at some point anyway.